intelmq/bots/outputs/smtp_batch/output.py

```diff
@@ -0,0 +1,355 @@
+# SPDX-FileCopyrightText: 2022 CSIRT.cz <https://csirt.cz>
+# SPDX-License-Identifier: AGPL-3.0-or-later
+import csv
+from dataclasses import dataclass
+import datetime
+import json
+import os
+import sys
+from tempfile import NamedTemporaryFile
+import time
+from typing import Any, Optional
+import zipfile
+from base64 import b64decode
+from collections import namedtuple, OrderedDict
```

Comment:
```suggestion
from collections import OrderedDict
```
`namedtuple` is not used

Reply:
Ok!